guacamole/src/main/frontend/src/app/auth/service/authenticationService.js

```diff
@@ -160,6 +160,8 @@ angular.module('auth').factory('authenticationService', ['$injector',
             url: 'api/tokens',
             headers: {
                 'Content-Type': 'application/x-www-form-urlencoded'
+                // FIXME: Clear current token if invalid, send token as header
```

Comment:
Is this still in need of fixing?

Reply:
Oops - nope, I addressed that. Will remove.

Reply:
Removed via rebase.